src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
It seems there might be a typo in the file name for the background-image URL. The project structure indicates the file should be `star_desactived.png` instead of `star.svg`. Please verify the correct file name and update accordingly.